requirements.txt

```diff
@@ -1,4 +1,4 @@
-python-telegram-bot
-apscheduler
-emoji
-xlsxwriter
+python-telegram-bot==9.0.0
+apscheduler==3.5.0
+emoji=0.4.5
```

Comment:
s/emoji=0.4.5/emoji==0.4.5/g

Btw, thanks bro for your contribution ;)

Reply:
ok. I will update this.